dvc/info.py

```diff
@@ -32,6 +33,10 @@ def get_dvc_info():
 
     try:
         with Repo() as repo:
+            hardlink_lock = repo.config["core"].get("hardlink_lock", False)
+            updater = Updater(repo.tmp_dir, hardlink_lock=hardlink_lock)
+            updater.check()
+
```

Comment:
The version info on the bottom makes more sense in the `version` output, like pip.
```console
...
You are using pip version 21.1.2; however, version 21.1.3 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.
```

Reply:
On a second thought, both box and free-floating text (btw, red is a  color of errors in my mind, probably not the best for upgrade notes) look out of place in `dvc version`'s current output. Let's just keep the old box for now, since it looks like `dvc version` output needs to be reconsidered.

Reply:
> btw, red is a color of errors in my mind

red is just a suggestion, I don't see why it cannot be yellow-ish.

> look out of place in dvc version's current output

This is what this PR is about right? free-floating text is much simpler than the box and fits with the version output more than the box. 

> Plus upgrading instructions (esp multiline ones) don't quite fit in pip-like message.

Multiline is only a problem for the binary package, and even then, it could be much simpler:
```\n
To upgrade, uninstall `dvc` and reinstall from https://dvc.org.
```

> since it looks like dvc version output needs to be reconsidered.

I don't see how this is related. `updater` message was already there if it fits or not. I don't see how version output needs to be considered to accommodate this, I consider them to be separate things.